README.md

```diff
@@ -1,6 +1,7 @@
 ## esday
 
-![npm](https://img.shields.io/npm/v/esday) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://makeapullrequest.com)
+[![npm](https://img.shields.io/npm/v/esday)](https://www.npmjs.com/package/esday)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://makeapullrequest.com)
```

Comment:
IMHO we should better use https://github.com/g-mero/esday?tab=readme-ov-file#contribute as the target